src/main/java/seedu/address/logic/parser/AddressBookParser.java

```diff
@@ -47,9 +48,9 @@ public class AddressBookParser {
     /**
      * Parses user input into command for execution.
      *
-     * @param userInput full user input string
-     * @return the command based on the user input
-     * @throws ParseException if the user input does not conform the expected format
+     * @param userInput The full user input string.
+     * @return the command based on the user input.
```

Comment:
Should be capital letter "The command based on the user input."

Reply:
resolved